docs/source/distributions/self_hosted_distro/starter.md

```diff
@@ -199,6 +199,19 @@ llama stack run distributions/starter/run.yaml \
   --env TOGETHER_API_KEY=your_together_key
 ```
 
+### Via venv
+
+Make sure you have done `uv pip install llama-stack` and have the Llama Stack CLI available.
+
+```bash
+llama stack build --template starter --image-type venv
```

Comment:
you can just do a single command with `uv` right?

```
 ENABLE_OLLAMA=ollama ENABLE_FIREWORKS=fireworks \
 OPENAI_API_KEY=... FIREWORKS_API_KEY=.... \
   uv run --with llama-stack llama stack build --template starter --image-type venv --run
```

Reply:
you still need to add `ENABLE_OLLAMA` etc. to the instructions. if the rest are also wrong, then update those too please.

Reply:
I reworked this and did the following:

1. Combined Conda and venv into one section (they are similar enough)
2. Prepended both with `uv run` (on my local machine the `--with llama-stack` was unnecessary even with a fresh `.venv`)
3. Didn't include any env vars as those are explained above, but added a note saying to ensure they are set before running.